### PR TITLE
[ADD] inventory: dispatch management system

### PR DESCRIPTION
**Pull request by Felicious**
Add new doc about a new feature available in 17.4+ used to plan and build deliveries.

Configuration instructions include:
- Fleet and vehicle setup
- Inventory configuration setup
- Creating and configuring batches
- New ways of creating batches using the **Transport Management** drop-down

Also includes a workflow of creating a batch and assigning it to a carrier vehicle. Then, show the maps and Google maps route to delivery items.

[Task](https://www.odoo.com/odoo/project/3835/tasks/4268459)

### CODE REVIEW

**Comment by Felicious** [comment]
Hello @larm-odoo ! I'm assigning you to review this PR since you're the Fleet expert on our team :blush: Looking forward to your review :heart_eyes:

File: content/applications/inventory_and_mrp/inventory/shipping_receiving/setup_configuration/dispatch.rst
Comment (larm-odoo) [inline content/applications/inventory_and_mrp/inventory/shipping_receiving/setup_configuration/dispatch.rst:171]:
```suggestion
<inventory/shipping_receiving/delivery-method-for-carrier> `.  A 
:guilabel:`Confirmation` pop-up window appears, indicating the number of orders being added to the batch. Click :guilabel:`Confirm` and the carrier is updated for all the selected records.
```

File: content/applications/inventory_and_mrp/inventory/shipping_receiving/setup_configuration/dispatch.rst
Comment (larm-odoo) [inline content/applications/inventory_and_mrp/inventory/shipping_receiving/setup_configuration/dispatch.rst:186]:
```suggestion
With the carrier set, add the desired orders to a batch by ticking the checkbox next to each order. Then, click
the :icon:`fa-cog` :guilabel:`Actions` button and click either :guilabel:`Add to batch` or
:guilabel:`Add to wave` to group the orders. In the resulting pop-up window, ensure the
:guilabel:`Add to` field is set to :guilabel:`a new [batch/wave] transfer`, and click
```
I would add a note directly after the first sentence to explain that the ticked orders that had the carrier applies remain ticked. And that ONLY if they wanted to add more orders, they need to tick those additional boxes (but they should add the carrier). Just a thought- since they would typically be adding what they just set the carrier to. I just noticed they stayed ticked which seemed unusual, so you can expand on that a touch.

File: content/applications/inventory_and_mrp/inventory/shipping_receiving/setup_configuration/dispatch.rst
Comment (larm-odoo) [inline content/applications/inventory_and_mrp/inventory/shipping_receiving/setup_configuration/dispatch.rst:246]:
```suggestion
or wave form, click the :guilabel:`Map` button. Doing so opens another page, displaying a map. Selecting an
```

Comment (larm-odoo) [reply content/applications/inventory_and_mrp/inventory/shipping_receiving/setup_configuration/dispatch.rst:246]: Just a note- mine did not open another tab or page- it just went to the map. Might want to double-check and change accordingly.

File: content/applications/inventory_and_mrp/inventory/shipping_receiving/setup_configuration/dispatch/google-map.png
Comment (larm-odoo) [inline content/applications/inventory_and_mrp/inventory/shipping_receiving/setup_configuration/dispatch/google-map.png:1]: My ONLY concern is- do we need to blur out the addresses since they are real addresses and not dummy/fake ones? I know that's more of a Youtube issue, but I honestly don't know what the policy is for documentation.  Just thinking out loud!

**Review by larm-odoo — APPROVED** [review]
Hi @Felicious - this was SUCH a cool document! And SUPER important for me to learn since I need to update my fleet docs with this info =).

Seriously, absolutely fascinating, and I learned so much.  I just had a bunch of small edits- noting major at all. Approving now!

File: content/applications/inventory_and_mrp/inventory/shipping_receiving/setup_configuration/dispatch.rst
Comment (Felicious) [reply content/applications/inventory_and_mrp/inventory/shipping_receiving/setup_configuration/dispatch.rst:39]: I'm not sure how to change these units of measure :see_no_evil:

File: content/applications/inventory_and_mrp/inventory/shipping_receiving/setup_configuration/dispatch.rst
Comment (Felicious) [reply content/applications/inventory_and_mrp/inventory/shipping_receiving/setup_configuration/dispatch.rst:53]: Thank you for elaborating! :D

File: content/applications/inventory_and_mrp/inventory/shipping_receiving/setup_configuration/dispatch.rst
Comment (Felicious) [reply content/applications/inventory_and_mrp/inventory/shipping_receiving/setup_configuration/dispatch.rst:186]: Thank you! this is a helpful note to add

File: content/applications/inventory_and_mrp/inventory/shipping_receiving/setup_configuration/dispatch.rst
Comment (Felicious) [reply content/applications/inventory_and_mrp/inventory/shipping_receiving/setup_configuration/dispatch.rst:35]: While this helps users find the button easier, according to @samueljlieber , we won't be including the section title of the dropdowns anymore. So I'll be ignoring this suggestion. Thank you though, Lara :blush: I'll consider this when the drop-downs contain a LOT more items, like the Technical menu.

File: content/applications/inventory_and_mrp/inventory/shipping_receiving/setup_configuration/dispatch/google-map.png
Comment (Felicious) [reply content/applications/inventory_and_mrp/inventory/shipping_receiving/setup_configuration/dispatch/google-map.png:1]: Hm.. that's a good concern. For the warehouse, I used the office's address, and picked random businesses on google maps. It's not someone's residence. Hopefully that makes it okay!

File: content/applications/inventory_and_mrp/inventory/shipping_receiving/setup_configuration/dispatch.rst
Comment (samueljlieber) [reply content/applications/inventory_and_mrp/inventory/shipping_receiving/setup_configuration/dispatch.rst:35]: Yes, specifically regarding this change in the guidelines 🙂 https://www.odoo.com/documentation/master/contributing/documentation/rst_guidelines.html#menu-selection

**Comment by Felicious** [comment]
Hello @odoo/us-doc-review ! This PR is ready for final review :D
Since it's a new feature, I asked the Inventory product expert, Sho, to do a once-over for accuracy, and he's approved it also, in addition to Lara :blush:

File: content/applications/inventory_and_mrp/inventory/shipping_receiving/setup_configuration/dispatch.rst
Comment (ksc-odoo) [inline content/applications/inventory_and_mrp/inventory/shipping_receiving/setup_configuration/dispatch.rst:97]:
```suggestion
To configure delivery methods, go to :menuselection:`Inventory app --> Configuration --> Delivery:
Delivery Methods`. Select an existing delivery method, or click :guilabel:`New`.
```

File: content/applications/inventory_and_mrp/inventory/shipping_receiving/setup_configuration/dispatch.rst
Comment (ksc-odoo) [inline content/applications/inventory_and_mrp/inventory/shipping_receiving/setup_configuration/dispatch.rst:104]:
```suggestion
use identifying information, such as the vehicle description and license plate number (e.g.
`Truck 123-ABCD`).
```

File: content/applications/inventory_and_mrp/inventory/shipping_receiving/setup_configuration/dispatch.rst
Comment (ksc-odoo) [inline content/applications/inventory_and_mrp/inventory/shipping_receiving/setup_configuration/dispatch.rst:128]:
```suggestion
:menuselection:`Inventory app --> Configuration --> Warehouse Management: Locations`.
```

File: content/applications/inventory_and_mrp/inventory/shipping_receiving/setup_configuration/dispatch.rst
Comment (ksc-odoo) [inline content/applications/inventory_and_mrp/inventory/shipping_receiving/setup_configuration/dispatch.rst:130]:
```suggestion
Click the desired location, which opens the :guilabel:`Location` form. In the :guilabel:`Additional
```

File: content/applications/inventory_and_mrp/inventory/shipping_receiving/setup_configuration/dispatch.rst
Comment (ksc-odoo) [inline content/applications/inventory_and_mrp/inventory/shipping_receiving/setup_configuration/dispatch.rst:141]:
```suggestion
Once setup is complete, :ref:`assign orders to a carrier
```

File: content/applications/inventory_and_mrp/inventory/shipping_receiving/setup_configuration/dispatch.rst
Comment (ksc-odoo) [inline content/applications/inventory_and_mrp/inventory/shipping_receiving/setup_configuration/dispatch.rst:144]:
```suggestion
<inventory/shipping_receiving/batch-form>`, as needed.
```

File: content/applications/inventory_and_mrp/inventory/shipping_receiving/setup_configuration/dispatch.rst
Comment (ksc-odoo) [inline content/applications/inventory_and_mrp/inventory/shipping_receiving/setup_configuration/dispatch.rst:146]:
```suggestion
Since this article is about a specific use case, explore details about each picking method in their
```

File: content/applications/inventory_and_mrp/inventory/shipping_receiving/setup_configuration/dispatch.rst
Comment (ksc-odoo) [reply content/applications/inventory_and_mrp/inventory/shipping_receiving/setup_configuration/dispatch.rst:155]: I agree with this suggestion

File: content/applications/inventory_and_mrp/inventory/shipping_receiving/setup_configuration/dispatch.rst
Comment (ksc-odoo) [inline content/applications/inventory_and_mrp/inventory/shipping_receiving/setup_configuration/dispatch.rst:162]:
```suggestion
Reveal the *Carrier* column, if it is not visible by default, by clicking the
```

File: content/applications/inventory_and_mrp/inventory/shipping_receiving/setup_configuration/dispatch.rst
Comment (ksc-odoo) [inline content/applications/inventory_and_mrp/inventory/shipping_receiving/setup_configuration/dispatch.rst:163]: this seems like it exceeds the 100 character limit

File: content/applications/inventory_and_mrp/inventory/shipping_receiving/setup_configuration/dispatch.rst
Comment (ksc-odoo) [inline content/applications/inventory_and_mrp/inventory/shipping_receiving/setup_configuration/dispatch.rst:166]:
```suggestion
   Other useful columns to enable can be :guilabel:`Zip` code, :guilabel:`Shipping Weight`, and
```

File: content/applications/inventory_and_mrp/inventory/shipping_receiving/setup_configuration/dispatch.rst
Comment (ksc-odoo) [inline content/applications/inventory_and_mrp/inventory/shipping_receiving/setup_configuration/dispatch.rst:173]:
```suggestion
batch. Click :guilabel:`Confirm`, and the carrier is updated for all the selected records.
```

File: content/applications/inventory_and_mrp/inventory/shipping_receiving/setup_configuration/dispatch.rst
Comment (ksc-odoo) [inline content/applications/inventory_and_mrp/inventory/shipping_receiving/setup_configuration/dispatch.rst:188]:
```suggestion
  If a delivery order is already assigned to a batch transfer, assigning a batch transfer here does
```

File: content/applications/inventory_and_mrp/inventory/shipping_receiving/setup_configuration/dispatch.rst
Comment (ksc-odoo) [inline content/applications/inventory_and_mrp/inventory/shipping_receiving/setup_configuration/dispatch.rst:192]:
```suggestion
or :guilabel:`Add to wave`. In the pop-up window, ensure the :guilabel:`Add to` field is set to :guilabel:`a
```

File: content/applications/inventory_and_mrp/inventory/shipping_receiving/setup_configuration/dispatch.rst
Comment (ksc-odoo) [inline content/applications/inventory_and_mrp/inventory/shipping_receiving/setup_configuration/dispatch.rst:178]: I could be mistaken, but I'm not sure if GUIs appear in figure captions

File: content/applications/inventory_and_mrp/inventory/shipping_receiving/setup_configuration/dispatch.rst
Comment (ksc-odoo) [inline content/applications/inventory_and_mrp/inventory/shipping_receiving/setup_configuration/dispatch.rst:203]:
```suggestion
Another place to create batches is by going to the :menuselection:`Inventory app`, and in the
```

File: content/applications/inventory_and_mrp/inventory/shipping_receiving/setup_configuration/dispatch.rst
Comment (ksc-odoo) [inline content/applications/inventory_and_mrp/inventory/shipping_receiving/setup_configuration/dispatch.rst:212]: Where is this located. Some minor elaboration on this would be helpful.

File: content/applications/inventory_and_mrp/inventory/shipping_receiving/setup_configuration/dispatch.rst
Comment (ksc-odoo) [inline content/applications/inventory_and_mrp/inventory/shipping_receiving/setup_configuration/dispatch.rst:218]:
```suggestion
   - :guilabel:`Manage Batches`: open list of batches
   - :guilabel:`Dock Dispatching`: open weekly calendar view of scheduled batch operations
   - :guilabel:`Batches by Route`: kanban view of batches grouped by fulfillment route
   - :guilabel:`Calendar`: open hourly calendar view of scheduled operations
   - :guilabel:`Statistics`: open pivot table of the batch transfers
```

**Review by ksc-odoo — APPROVED** [review]
Hey @Felicious -- just finished my Final Review of this PR. Nice job. Once you address my feedback, and implement the necessary adjustments, feel free to tag this for Tech Review. Thanks!

File: content/applications/inventory_and_mrp/inventory/shipping_receiving/setup_configuration/dispatch.rst
Comment (Felicious) [reply content/applications/inventory_and_mrp/inventory/shipping_receiving/setup_configuration/dispatch.rst:128]: Hi @ksc-odoo !

According to the [content guidelines](https://www.odoo.com/documentation/16.0/contributing/documentation/content_guidelines.html#lists),  it seems that section names are no longer included when navigating through menu items, and we're now listing only the items themselves. Could you confirm if this is correct?

cc: @samueljlieber

File: content/applications/inventory_and_mrp/inventory/shipping_receiving/setup_configuration/dispatch.rst
Comment (Felicious) [reply content/applications/inventory_and_mrp/inventory/shipping_receiving/setup_configuration/dispatch.rst:155]: Hi @ksc-odoo !

According to the [content guidelines](https://www.odoo.com/documentation/16.0/contributing/documentation/content_guidelines.html#lists),  it seems that section names are no longer included when navigating through menu items, and we're now listing only the items themselves. Could you confirm if this is correct?

cc: @samueljlieber

File: content/applications/inventory_and_mrp/inventory/shipping_receiving/setup_configuration/dispatch.rst
Comment (Felicious) [reply content/applications/inventory_and_mrp/inventory/shipping_receiving/setup_configuration/dispatch.rst]: This is the menu that I referred to in the line right before this. I'm not sure if elaboration is necessary here, unless the user cannot see the image.

**Comment by Felicious** [comment]
Thank you for the thorough review, @ksc-odoo! 
Tagging @odoo/us-doc-review for tech review (:

File: content/applications/inventory_and_mrp/inventory/shipping_receiving/setup_configuration/dispatch.rst
Comment (samueljlieber) [inline content/applications/inventory_and_mrp/inventory/shipping_receiving/setup_configuration/dispatch.rst:19]: Suggestion to link to the install doc :)
```suggestion
#. :ref:`Install <general/install>` the **Fleet** application.
```

File: content/applications/inventory_and_mrp/inventory/shipping_receiving/setup_configuration/dispatch.rst
Comment (samueljlieber) [inline content/applications/inventory_and_mrp/inventory/shipping_receiving/setup_configuration/dispatch.rst:40]: UI field name is "Max Volume" and remove extra backtick :)
```suggestion
:guilabel:`Max Volume` (in cubic meters) fields.
```

File: content/applications/inventory_and_mrp/inventory/shipping_receiving/setup_configuration/dispatch.rst
Comment (samueljlieber) [inline content/applications/inventory_and_mrp/inventory/shipping_receiving/setup_configuration/dispatch.rst:53]: I suggest swapping the order of these elements, so that the relevant image follows the paragraph, then the note admonition, then the seealso. Otherwise, the seealso seems to break the flow and image loses some context IMO
```suggestion

.. image:: dispatch/categories.png
   :alt: Vehicle categories with defined weight and volume.

.. note::
   The units of measurement for vehicle capacity are assigned at the global level in the
   **Settings** app, in the :guilabel:`Units of Measure` section.

   .. image:: dispatch/settings.png
      :alt: Show units of measure settings.

.. seealso::
   :ref:`Vehicle category <fleet/categories>`

```

File: content/applications/inventory_and_mrp/inventory/shipping_receiving/setup_configuration/dispatch.rst
Comment (samueljlieber) [inline content/applications/inventory_and_mrp/inventory/shipping_receiving/setup_configuration/dispatch.rst:69]: Again, suggesting to swap the seealso position to come after the paragraph :)
```suggestion
From the :guilabel:`Models` list, select an existing model, or click :guilabel:`New` in the top-left
corner to create a new model. Then, set the relevant :guilabel:`Category` field to the relevant
vehicle category.

.. seealso::
   :doc:`Create car model <../../../../hr/fleet/models>`

```

File: content/applications/inventory_and_mrp/inventory/shipping_receiving/setup_configuration/dispatch.rst
Comment (samueljlieber) [inline content/applications/inventory_and_mrp/inventory/shipping_receiving/setup_configuration/dispatch.rst:94]: Just double checking- it isnt stated anywhere in this section whether or not to publish the newly created/configured delivery method. Is it necessary to state this?

File: content/applications/inventory_and_mrp/inventory/shipping_receiving/setup_configuration/dispatch.rst
Comment (samueljlieber) [inline content/applications/inventory_and_mrp/inventory/shipping_receiving/setup_configuration/dispatch.rst:138]: Figure attribute alignment :)
```suggestion
.. figure:: dispatch/dock-location.png
   :alt: Location configuration.

   Location configuration page with :guilabel:`Is a Dock Location` checkbox ticked.
```

File: content/applications/inventory_and_mrp/inventory/shipping_receiving/setup_configuration/dispatch.rst
Comment (samueljlieber) [inline content/applications/inventory_and_mrp/inventory/shipping_receiving/setup_configuration/dispatch.rst:157]: Another suggestion to swap seealso position and text, just to avoid breaking the flow of the paragraph
```suggestion
To group products, go to the :menuselection:`Inventory app --> Operations --> Deliveries`, which
reveals a list of outgoing deliveries.

.. seealso::
   Since this article is about a specific use case, explore details about each picking method in
   their dedicated articles.

   - :doc:`../picking_methods/batch`
   - :doc:`Wave picking <../picking_methods/wave>`
   - :doc:`../picking_methods/cluster`

```

File: content/applications/inventory_and_mrp/inventory/shipping_receiving/setup_configuration/dispatch.rst
Comment (samueljlieber) [inline content/applications/inventory_and_mrp/inventory/shipping_receiving/setup_configuration/dispatch.rst:165]: Missing a "p" ;)
```suggestion
:icon:`oi-settings-adjust` :guilabel:`(settings)` icon in the top-right corner, and ticking the
```

File: content/applications/inventory_and_mrp/inventory/shipping_receiving/setup_configuration/dispatch.rst
Comment (samueljlieber) [inline content/applications/inventory_and_mrp/inventory/shipping_receiving/setup_configuration/dispatch.rst:215]: Please capitalize "Kanban" :)
```suggestion
   - :guilabel:`Batches by Route`: Kanban view of batches grouped by fulfillment route
```

File: content/applications/inventory_and_mrp/inventory/shipping_receiving/setup_configuration/dispatch.rst
Comment (samueljlieber) [inline content/applications/inventory_and_mrp/inventory/shipping_receiving/setup_configuration/dispatch.rst:220]: Image alt alignment and small alt tag change to be more descriptive. 

Also, I'd like to see this image cropped to just the delivery orders card, IMO there is no need to show the whole UI here :)

Screenshot 2024-10-22 at 11 30 55 AM

```suggestion
.. image:: dispatch/prepare-batch.png
   :alt: Show prepare batch option from the Transport Management drop-down menu.
```

**Review by samueljlieber — APPROVED** [review]
Hi @Felicious, great job on this brand new Inventory doc on the Dispatch management system! This is really interesting and I learned a lot! I like how you organized this doc too!

I am approving with a couple fixes, and some suggestions for your consideration. Thank you for your work!

.....
@robodoo delegate=Felicious

File: content/applications/inventory_and_mrp/inventory/shipping_receiving/setup_configuration/dispatch.rst
Comment (Felicious) [reply content/applications/inventory_and_mrp/inventory/shipping_receiving/setup_configuration/dispatch.rst:94]: Thanks for being thorough :blush: Publishing the delivery method is necessary if it's available on the website. I don't think it's necessary to mention this, since it's more related to ecommerce stuff and is mentioned on the delivery method doc

**Comment by Felicious** [comment]
@robodoo r+

**Pull request closed without merging**
